app/views/admin/users/_member_index.html.erb

```diff
@@ -7,34 +7,62 @@
+                <a href="<%= admin_user_path(user.id) %>">
+                  <img class="radius-full align-middle" src="<%= user.profile_image_url_for(length: 50) %>" width="40" height="40" alt="<%= user.name %>" />
+                </a>
+                <div class="pl-3 flex-1 fs-s"><h3 class="fs-base"><%= link_to user.name, admin_user_path(user) %></h3>
+                  <p class="color-base-60"><%= link_to "@#{user.username}", admin_user_path(user) %>&nbsp;
+                  </p>
                 </div>
               </header>
               <div class="flex justify-between py-2">
-                <div>standing info</div>
-                <div>role info</div>
+                <div>
+                <% if user.suspended? %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #DC2626"></span>Suspended
+                <% elsif user.warned? %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #F59E0B"></span>Warned
+                <% elsif user.comment_suspended? %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #DC2626"></span>Comment suspended
+                <% elsif user.trusted? %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #059669"></span>Trusted
+                <% else %>
+                  <span class="c-indicator mr-2 inline-block" style="--bg: #A3A3A3"></span>Good standing
+                <% end %>
+                </div>
+                <div>
+                <span class="c-indicator ml-2 crayons-hover-tooltip" data-tooltip="<%= user.current_role %>">
+                  <%= user.current_role %>
+                </span>
+                </div>
               </div>
               <div class="flex justify-between items-end">
                 <dl class="flex gap-2">
                   <div>
                     <dt class="color-base-60 fw-normal">Last activity</dt>
-                    <dd>placeholder</dd>
+                    <dd><%= user.latest_activity %></dd>
                   </div>
                   <div>
                     <dt class="color-base-60 fw-normal">Joined on</dt>
                     <dd><%= user.registered_at.strftime("%d %b, %Y") %></dd>
                   </div>
                 </dl>
-                <div>Orgs info</div>
+                <div>
+                  <% if user.organization_memberships.any? %>
+                    <% user.organization_memberships.includes([:organization]).each do |org_membership| %>
+                      <span class="crayons-hover-tooltip" data-tooltip="<%= org_membership.organization.name %>">
+                        <img src="<%= org_membership.organization.profile_image_url_for(length: 64) %>" width="32" height="32" alt="<%= org_membership.organization.name %>" class="c-link crayons-logo crayons-logo--l">
+                      </span>
+                    <% end %>
+                  <% else %>
+                    -
```

Comment:
(not blocking) I just realised in the mobile designs I don't think we have an example of someone with no organisations... I wonder if it would make more sense to just leave a blank space in this case rather than `-`, since there's no column header to explain to a user what would otherwise be in this space.

Anyway, this is grand for now, just a thought for future!

Reply:
✔️